unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarRenderer.cs

```diff
@@ -302,12 +308,19 @@ private IEnumerator LoadAvatar()
 
             yield return new WaitUntil(() => bodyShapeController.isReady && wearableControllers.Values.All(x => x.isReady));
 
+
             eyesController.Load(bodyShapeController, model.eyeColor);
             eyebrowsController.Load(bodyShapeController, model.hairColor);
             mouthController.Load(bodyShapeController, model.skinColor);
 
             yield return new WaitUntil(() => eyebrowsController.isReady && eyesController.isReady && mouthController.isReady);
 
+            if (useFx && (bodyIsDirty || wearablesIsDirty))
+            {
+                Instantiate(fxSpawnPrefab).transform.position += transform.position;
+                Instantiate(fxChangeWearablePrefab).transform.position += transform.position;
```

Comment:
Only by curiossity (maybe it is designed in this way): Are both effects (spawn and wearable change) always run together? Even if an user is already loaded into the world and only change the wearable? In this case, shouldn't we see only the wearable change fx?